elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/EntityProjectionTranslator.java

```diff
@@ -46,13 +46,17 @@
     private FilterExpression whereFilter;
     private FilterExpression havingFilter;
     private EntityDictionary dictionary;
+    private Boolean bypassCache;
+
 
     public EntityProjectionTranslator(QueryEngine engine, Table table,
-                                      EntityProjection entityProjection, EntityDictionary dictionary) {
+                                      EntityProjection entityProjection, EntityDictionary dictionary,
+                                      String bypassCache) {
         this.engine = engine;
         this.queriedTable = table;
         this.entityProjection = entityProjection;
         this.dictionary = dictionary;
+        this.bypassCache  = (bypassCache != null && bypassCache.equals("true")) ? true : false;
```

Comment:
This should move into the calling code.  It is a strange contract to accept a string header here.